config/base_config.go

```diff
@@ -50,6 +50,9 @@ type BaseConfig struct {
 	EventDispatcherType string        `default:"direct" yaml:"event_dispatcher_type" json:"event_dispatcher_type,omitempty"`
 	MetricConfig        *MetricConfig `yaml:"metrics" json:"metrics,omitempty"`
 	fileStream          *bytes.Buffer
+
+	// cache file used to store the current used configurations.
+	CacheFile string `yaml:"cache_file" json:"cache_file,omitempty" property:"cache_file"`
```

Comment:
I can't find where to use this field.

Reply:
Please see `config_loader.go` file.

Reply:
Yes, you write provider config to CacheFile, but I want to know where to use it.

Reply:
Please see the issue comment: #729 (comment).